scripts/after-shipit-pod-push.js

```diff
@@ -6,10 +6,17 @@ const { execSync } = require('child_process');
 class AfterShipItPodPush {
   name = 'after-shipit-pod-push';
 
+  wasCanary = false;
+
   apply(auto) {
+    auto.hooks.canary.tap(this.name, () => {
```

Comment:
This looks good to me. Only thought is if it would make sense just to early return if we `release` is `undefined` and log? Just trying to think if there are any other use cases where `afterShipIt` would be invoked but not have an actual release.

Reply:
AFAIK theres only 3 flows, release, prerelease and canary? and the first two we do have tags, so if we get the canary hook called, i can skip even hitting the release API since i know it wont have a tag